worker/pkg/benthos/transformers/data-sets/data_generator.go

```diff
@@ -68,7 +68,7 @@ func main() {
 
 	for _, line := range lines {
 		trimmedLine := strings.TrimSpace(line)
-		if len(trimmedLine) == 0 || strings.ContainsAny(trimmedLine, " -_") {
+		if len(trimmedLine) == 0 || strings.ContainsAny(trimmedLine, "-_") {
```

Comment:
Hm, I'm guessing there was a reason why I originally wrote this to exclude first/last names with spaces. But who knows if that is still relevant. Part of me wants to expose any bugs removing the space here would cause...